src/Components/content.css

```diff
@@ -0,0 +1,9 @@
+.Twitter-toggle {
```

Comment:
Why did you create 3 classes? Isn't it enough to create 1 class called .hidden and use it in any element that you want to hide? :)